src/main.rs

```diff
@@ -706,6 +755,30 @@ fn get_clap_argument(arg_name: &str) -> Result<Arg> {
             .value_name("subject_did")
             .help("DID of subject")
             .takes_value(true),
+        "operation" => Arg::with_name("operation")
+            .long("operation")
+            .value_name("operation")
+            .help("Operation type AddKey/RemoveKey/AddServiceEnpoint/RemoveServiceEnpoint")
+            .takes_value(true),
+        "update_key" => Arg::with_name("update_key")
+            .long("update_key")
+            .value_name("update_key")
+            .help("Update key for did update in JWK format")
+            .takes_value(true),
+        "bbs_key" => Arg::with_name("bbs_key")
+            .long("bbs_key")
+            .value_name("bbs_key")
+            .help("pre-generated bbs public key")
+            .takes_value(true),
+        "signing_key" => Arg::with_name("signing_key")
+            .long("signing_key")
+            .value_name("signing_key")
+            .help("signing key to be added to did doc")
+            .takes_value(true),
+        "service_endpoint" => Arg::with_name("service_endpoint")
+            .long("service_endpoint")
+            .value_name("service_endpoint")
+            .help("Service endpoint url"),
```

Comment:
These are not marked as mandatory and cli would show them as optional. When not providing them, `get_argument_value` would then panic. This would run correctly but we could avoid a bit of friction here if we just defined them as required. ^^

Reply:
marked required fields as required